games-puzzle/zaz/zaz-1.0.0.recipe

```diff
@@ -48,42 +49,34 @@ BUILD_REQUIRES="
 	devel:libtheoradec
 	devel:libtheoraenc
 	devel:libvorbisfile
-"
-
+	"
 BUILD_PREREQUIRES="
 	cmd:aclocal
 	cmd:autoconf
+	cmd:autoheader
 	cmd:automake
+	cmd:autoreconf
 	cmd:awk
 	cmd:find
 	cmd:gcc
 	cmd:make
 	cmd:pkg_config
-"
-
-DEPEND="
-        media-libs/sdl-image >= 1.2.10
-"
+	"
 
-BUILD() {
-	# Gettextize wants the user to press enter after running it, remove that
-	# "feature" from it...
+BUILD()
+{
+#	Gettextize wants the user to press enter after running it, remove that
+#	"feature" from it...
 	grep -v '/dev/tty' < /bin/gettextize > gettextize
 	chmod +x gettextize
 
-#libtoolize --force --copy --install
-	automake --add-missing
-	aclocal
+	autoreconf -vfi
 	./gettextize -f
```

Comment:
I wonder if this is still needed.

Reply:
It is :) version mismatch if I leave it out

Reply:
Still needed, without autoreconf it's using "-I/usr/include/freetype2" for freetype's headers